src/renderer/containers/Settings/DolphinSettings.tsx

```diff
@@ -1,15 +1,36 @@
 /** @jsx jsx */
+import { GeckoCode, makeGeckoCodeFromRaw } from "@dolphin/geckoCode";
 import {
+  ipc_addGeckoCode,
   ipc_clearDolphinCache,
   ipc_configureDolphin,
+  ipc_convertGeckoToRaw,
+  ipc_deleteGecko,
+  ipc_fetchGeckoCodes,
+  ipc_fetchSysInis,
   ipc_importDolphinSettings,
   ipc_reinstallDolphin,
+  ipc_toggleGeckos,
 } from "@dolphin/ipc";
 import { DolphinLaunchType } from "@dolphin/types";
 import { css, jsx } from "@emotion/react";
+import { Box, Tab, Tabs } from "@material-ui/core";
```

Comment:
You can't import components from material-ui this way without breaking things. Update the imports to be like the ones below.